Codespell does not ignore ./media directory

Codespell should ignore files from local development to prevent unnecessary errors such as:

```
➜  authentik git:(sdko/proxy-outpost-sqlite) ✗ make lint-fix
uv run codespell -w
WARNING: Cannot decode file using encoding "utf-8": ./media/public/application-icons/66226e02-5952-5e08-a512-33c82513ef74.pdf
WARNING: Trying next encoding "iso-8859-1"
./media/public/application-icons/66226e02-5952-5e08-a512-33c82513ef74.pdf:86: fO ==> of, for, to, do, go
./media/public/application-icons/66226e02-5952-5e08-a512-33c82513ef74.pdf:187: Fo ==> Of, For, To, Do, Go
make: *** [lint-codespell] Error 65
➜  authentik git:(sdko/proxy-outpost-sqlite) ✗
```

--- a/pyproject.toml
+++ b/pyproject.toml
@@ -150,6 +150,7 @@ skip = [
     "./gen-go-api",
     "*.api.mdx",
     "./htmlcov",
+    "./media",
 ]
 dictionary = ".github/codespell-dictionary.txt,-"
 ignore-words = ".github/codespell-words.txt"

